src/python/espressomd/shapes.py

```diff
@@ -22,6 +22,28 @@ class Cylinder(ScriptInterfaceHelper):
     """
     _so_name = "Shapes::Cylinder"
 
+@script_interface_register
+class Ellipsoid(ScriptInterfaceHelper):
+    """
+    An ellipsoid.
+
+    For now only ellipsoids of revolution are supported.
+    The symmetry axis is aligned parallel to the x-direction.
+
+    Attributes
+    ----------
+    center : :obj:`array_like`
+       Coordinates of the center of the ellipsoid.
+    a : :obj:`float`
+       Semiaxis along the axis of rotational symmetry.
```

Comment:
An ellipsoid has to axes of rotational symmetry. Also please describe what the parameter is, since it is only a number and not an array describing an axis. In this case I would also opt for a longer variable name that is more descriptive.

Reply:
Same here.

Reply:
An ellipsoid of revolution has exactly one axis of rotational symmetry. Otherwise it is a sphere and all axis (`a`, `b`, `b`) are the same anyway.